spec/models/poll_spec.rb

```diff
@@ -4,9 +4,35 @@
   it { is_expected.to have_many :ballots }
   it { is_expected.to have_many :users }
 
-  let!(:polls) { FactoryGirl.create_list :poll, 2 }
+  let(:poll) { create :poll }
```

Comment:
There are a few places in this file where we're doing `create :thing` and `FactoryGirl.create :thing`. I'd prefer the latter.